.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report
+about: Report a bug
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Expected Behavior
+
+_Describe what you expected to happen._
+
+## Actual behavior
+
+_Describe what actually happened._
+
+## Steps to reproduce the issue
+
+_List the steps to reproduce the issue._
+
+_1. Step 1_
+
+_2. Step 2_
+
+...
+
+## Logs/Error Messages
+
+_Attach or paste appropriate logs and error messages to help us debug better._
+
+## Name and Version Information
+
+_Specify the name and version of the following components:_
+
+_* Development Environment (For example, XCode version 12.0)._
+
+_* SDK (For example, Omise-iOS version 4.25)._
+
+_* Package Manager (For example, Swift version 5.0)._
+
+_* Operating System (For example, Swift version 5.0)._
```

Comment:
This is not an example of an operating system.

Reply:
Rectified.